src/internal/docker/cli.go

```diff
@@ -33,7 +33,7 @@ func Run(ctx context.Context, yeyCtx yey.Context, containerName string, options
 		return startContainer(ctx, containerName, options)
 	case "running":
 		yey.Log("executing new shell in running container %q", containerName)
-		return execContainer(ctx, containerName, yeyCtx.Cmd, options)
+		return execContainer(ctx, yeyCtx, containerName, options)
```

Comment:
might be nicer to only pass the things we need rather than an entire `yey.Context`.

Reply:
meh nvm